### ISSUE TITLE
@Erased on all classes in a package still generates namespace in ts

### ISSUE THREAD

**Issue by StormeHawke** [cincheo/jsweet#270]
If you have @Erased on all classes in a package (e.g. `somepackage.jvm.validation` for classes used for custom jsr-303 validation classes only used in java), the namespace is still generated in the TS even though there are no TS classes generated by TS.  It's two lines of code including the comment so minor but would be best not to include

Currently generates the following namespace:
```
/* Generated from Java with JSweet 1.2.1 - http://www.jsweet.org */
namespace somepackage.jvm.validation {}
```

**Comment by renaudpawlak** [comment]
I am almost 100% sure that this has been fixed in master (2.0.0-SNAPSHOT).

**Comment by renaudpawlak** [comment]
I has been fixed but it has been undone. The right way to erase a package is to add the @Erased annotation on the package.
